embedded-graphics/src/geometry/real.rs

```diff
@@ -0,0 +1,214 @@
+use core::ops::{Add, AddAssign, Div, DivAssign, Mul, MulAssign, Neg, Sub, SubAssign};
+use float_cmp::{ApproxEq, F32Margin};
+#[cfg(not(feature = "fixed_point"))]
+#[allow(unused_imports)]
+use micromath::F32Ext;
+
+/// Real.
+///
+/// `Real` is used to store real numbers as either floating point or fixed
+/// point depending on feature flag.
```

Comment:
This should mention the feature in question:

```suggestion
/// `Real` is used to store real numbers as either floating point or 32 bit fixed point if the
/// `fixed_point` cargo feature is enabled.
```